File: src/com/funivan/phpstorm/refactoring/util/PhpIndexUtil.java
package com.funivan.phpstorm.refactoring.util;

import com.jetbrains.php.lang.psi.elements.Method;
import com.jetbrains.php.lang.psi.elements.PhpClass;
import org.jetbrains.annotations.Nullable;

import java.util.Collection;

/**
 * @author Ivan Shcherbak <alotofall@gmail.com>
 */
public class PhpIndexUtil {

    @Nullable
    public static Method getClassMethod(PhpClass phpClass, String methodName) {
        Collection<Method> classMethods = phpClass.getMethods();
        for (Method m : classMethods) {
            if (m.getName().equals(methodName)) {
                return m;
            }
        }

        return null;
    }

}

File: src/com/funivan/phpstorm/refactoring/util/PhpIndexUtil.kt
package com.funivan.phpstorm.refactoring.util

import com.jetbrains.php.lang.psi.elements.Method
import com.jetbrains.php.lang.psi.elements.PhpClass

/**
 * @author Ivan Shcherbak alotofall@gmail.com
 */
object PhpIndexUtil {

    fun getClassMethod(phpClass: PhpClass, methodName: String): Method? {
        val classMethods = phpClass.methods
        for (m in classMethods) {
            if (m.name == methodName) {
                return m
            }
        }

        return null
    }

}
